docker/package/tezos_setup_wizard.py

```diff
@@ -77,7 +77,7 @@
 def fetch_snapshot(url):
     print("Downloading the snapshot from", url)
     filename = TMP_SNAPSHOT_LOCATION
-    urllib.request.urlretrieve(url, filename, progressbar_hook)
+    proc_call(f"wget --show-progress --continue -O {filename} {url}")
```

Comment:
AFAIU there's no built-in way to do this with `urllib`, but it should also be implementable.

Have you considered doing that? If so, were there problems with it?

Reply:
I considered this, but it turns out to be much more verbose and I don't really see any benefit from it, `wget` interface and output seems to be very descriptive and download resume is working fine.

Reply:
OK, I'm not against it, but in this case `wget` should be listed as a run dependency for the `tezos-baking` package

Reply:
It turns out to be non-trivial to make properly with our code now, but since I'm reconsidering `tezos-baking` packaging now I'll figure out how to split binaries packaging (and its deps) and `tezos-baking` ones